tox.ini

```diff
@@ -20,13 +20,13 @@ norecursedirs = .* requirements
 
 [testenv]
 deps =
-    django18: Django>=1.8,<1.9
-    django19: Django>=1.9,<1.10
-    django110: Django>=1.10,<1.11
-    django111: Django>=1.11,<2
+    django111: Django>=1.11,<2.0
+    django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<2.2
+    django22: Django>=2.2,<2.3
     -r{toxinidir}/requirements/test.txt
 commands =
-    py.test {posargs}
+    py.test -Wd {posargs}
```

Comment:
`python -Wd -m pytest {posargs}` is a little better here, since it can display deprecation warnings triggered during pytest plugin initialization.